pkg/kritis/metadata/containeranalysis/containeranalysis_int_test.go

```diff
@@ -137,3 +140,30 @@ func TestCreateAttestationNoteAndOccurrence(t *testing.T) {
 		}
 	}
 }
+
+func TestGetMultiplePages_Vulnerabilities(t *testing.T) {
+	d, err := New()
+	if err != nil {
+		t.Fatalf("Could not initialize the client %s", err)
+	}
+
+	// Set PageSize to 300
+	createListOccurrencesRequest = createListOccurrencesRequestTest
+
+	vuln, err := d.Vulnerabilities("gcr.io/kritis-int-test/java-with-vulnz@sha256:358687cfd3ec8e1dfeb2bf51b5110e4e16f6df71f64fba01986f720b2fcba68a")
+	if err != nil {
+		t.Fatalf("Found err %s", err)
+	}
+
+	if len(vuln) <= 300 {
+		t.Errorf("Pagination error: only received results from first page.")
```

Comment:
t.Fatalf?

Reply:
I'm curious why t.Fatalf would be better here? My understanding was that the difference between t.Fatalf and t.Errorf is that the latter ends execution immediately. Would that matter here, as it is the last statement in the test?

Reply:
Yes, logically here it's same, but just as a consistency to the rest of the code. And we never know if the difference could play out elsewhere.

Reply:
Done.